src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -222,13 +222,12 @@ function MultiCombobox<Item>({
             onMouseDown: toggleMenu,
             readOnly: true,
             placeholder: !value.length ? placeholder : '',
-            position: !value.length ? 'static' : 'absolute',
           }),
           ...(searchable && {
-            placeholder,
-            mt: (isOpen && value.length) || (isOpen && value.length && allowAdditions) ? -4 : 0,
-            position: isOpen || (isOpen && value.length && allowAdditions) ? 'static' : 'absolute',
+            placeholder: hideLabel && value.length && !isOpen ? '' : placeholder,
```

Comment:
This fixes a visual bug when the label was hidden the input placeholder was rendered behind the Tags.
![image](https://user-images.githubusercontent.com/15084305/109523265-d4a32e80-7ab7-11eb-9da9-145dfc03e1fd.png)